apps/condo/domains/resident/schema/Resident.js

```diff
@@ -211,6 +212,13 @@ const Resident = new GQLListSchema('Resident', {
                 }
             }
         },
+        afterChange: async ({ updatedItem, operation }) => {
+            const { unitName, unitType, property: propertyId, user: userId } = updatedItem
+
+            if (operation === 'create') {
```

Comment:
We also use `update` to restore old Resident records. It's better to detect property/unitName/unitType changes rather than use opertation.